uint16 id
uint16 center           2048
uint16 range            4096
bool   invert_direction false
uint16 default_value    0
---
bool   successful true
string message    ""
